### DISCUSSION TITLE
question about data file access for IRB

### DISCUSSION THREAD

**Discussion by dickinsonca** [tatool/tatool-web#171] [General]
Hello,
I need to report information about data file security for an IRB application.  I'm assuming that if I am collecting data using a public project that is hosted on Tatool's web site, that I will be the only person who will be able to access participants' data.  Would someone please let me know if that is correct?  Thank you.

Chris Dickinson

**Comment by cvonbastian** [comment]
Hi Chris! Yes, as long as you don't share your user credentials with anyone, you'll be the only user who can access your participants' data. Of course though, as we're hosting tatool-web.com on our server, we generally also have access as the processing and storing of the data is happening via the Google Cloud Platform, and we need access for maintenance and administration (similar to how Qualtrics etc. also generally will have access to user data). The data are stored on a EU server in case that's important for your IRB too. 

In case you need to have completely exclusive access to your data, you'd need to run a local Tatool Web instance in your lab, so that all data are stored only locally but not on the web.
